indigo_app/static/stylesheets/app.scss

```diff
@@ -39,6 +39,7 @@ $fa-font-path: "../lib/fontawesome-5.2/webfonts";
 @import 'utilities';
 @import 'works';
 @import 'provisions-list';
+@import "treeview";
```

Comment:
Why create a new set of styles, instead of adjusting the provisions-list styles, which you added just for this use case IIRC? Don't we now run the risk that `provisions-list` is never used?

Reply:
To answer my own question: because they're different things!